azure-pipelines.yml

```diff
@@ -22,7 +22,7 @@ resources:
   - repository: das-platform-building-blocks
     type: github
     name: SkillsFundingAgency/das-platform-building-blocks
-    ref: refs/tags/0.4.29
+    ref: refs/tags/0.4.30
```

Comment:
```suggestion
    ref: refs/tags/0.4.34
```

Reply:
@jack-cloud-platform Thanks Jack, I have now updated the version from ref: refs tags 0.4.30  to 34